crowbar_framework/app/models/backup.rb

```diff
@@ -75,14 +75,19 @@ def restore(options = {})
     from_upgrade = options.fetch(:from_upgrade, false)
 
     if Crowbar::Backup::Restore.restore_steps_path.exist?
+      logger.debug "Restore process is already running"
       errors.add(:base, I18n.t("backups.index.multiple_restore"))
       return false
     elsif !from_upgrade && backup_version != system_version
+      logger.debug "Cannot restore from different Crowbar version"
```

Comment:
i just took the i18n strings that we are already using, so shall we change them as well?